api/NodeList.json

```diff
@@ -271,10 +271,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "≤4"
```

Comment:
Can you set this to 1 to match the parent feature? This interface makes no sense without the `length` property.